### ISSUE TITLE
[Bug] KopiaUI not failing gracefully if cannot connect to repository

### ISSUE THREAD

**Issue by basldfalksjdf** [kopia/kopia#2069]
I have KopiaUI set to backup to a shared NAS. However, I only have access to this NAS on a particular network, so when I am on another network, Kopia cannot connect to the NAS. When this happens, KopiaUI is just stuck at the "Loading..." screen (see screenshot below). 

Is there anyway to make it fail gracefully, where the user is displayed a message that Kopia cannot connect to repo and give users the option to disconnect the repo if desired or allow KopiaUI to keep trying to connect (in which case it will start the backups again when the repo becomes available).

![Capture](https://user-images.githubusercontent.com/106879934/174835739-c9ea73fb-9759-4b9b-852a-9766042ba07c.PNG)

**Comment by adowair** [comment]
Just a sanity thought: is this a reflection of getting stuck in some connection backoff loop? What is the behavior of the CLI? Does that also hang up, and if not, when does an error message come back?

**Comment by basldfalksjdf** [comment]
I don't use the CLI, but I tried `kopia repository connect filesystem --path //path/to/nas/folder` and I got the error `kopia.exe: error: can't connect to storage: filesystem repository path must be absolute, try --help`. I suspect, though, that this error may be different than what KopiaUI is experiencing, because backing up to the local NAS via KopiaUI works just fine when I am on the appropriate local network.

Also, from what I can tell, KopiaUI is not "frozen", and I do not need to forcequit it. It is just stuck at the loading screen.
